views/app.ejs

```diff
@@ -329,6 +331,8 @@
 			var app = new Wu.App({
 				id : 'app'
 			});
+			
+			app.raven();
```

Comment:
- [ ] This should be run from inside Wu.App. We don't want more js code here, wanna keep it as neat as possible.
- [ ] You can also remove old raven script in line 132.